timescaledb/how-to-guides/compression/compression.md

```diff
@@ -1,4 +1,4 @@
-# Compression basics
+# Compression
```

Comment:
To explain this change: compression is anything *but* basic. By calling it basic, we're essentially telling any reader who doesn't understand it that they're a bit stupid 😉